tests/components/add_user_to_channel_modal.test.jsx

```diff
@@ -0,0 +1,260 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {shallow} from 'enzyme';
+
+import AddUserToChannelModal from 'components/add_user_to_channel_modal/add_user_to_channel_modal';
+
+describe('components/AddUserToChannelModal', () => {
+    const baseProps = {
+        channelMembers: {},
+        user: {
+            id: 'someUserId',
+            first_name: 'Fake',
+            last_name: 'Person',
+        },
+        onHide: jest.fn(),
+        actions: {
+            addChannelMember: jest.fn(() => {
```

Comment:
There's a [warning on testing](https://build.mattermost.com/blue/organizations/jenkins/mw%2Fmattermost-webapp/detail/PR-1918/12/pipeline/29/#step-30-log-36) since no value was returned by `addChannelMember`.  You may resolve it by returning `{}` or by simplifying the mock up like,
```javascript
actions: {
    addChannelMember: jest.fn().mockResolvedValue({}), // if you'd like to return an error "mockResolvedValue({error: 'some error'})"
    getChannelMember: jest.fn().mockResolvedValue({}),
}
```

Reply:
Great, I saw this happen on a few other tests so I figured it may have been expected. Thanks for pointing it out!